src/plugins/discover/public/application/main/components/top_nav/open_alerts_popover.tsx

```diff
@@ -84,7 +85,7 @@ export function AlertsPopover({
 
     return triggersActionsUi?.getAddRuleFlyout({
       metadata: discoverMetadata,
-      consumer: 'discover',
```

Comment:
About this change we created an issue to the breaking changes committee -> https://github.com/elastic/dev/issues/2344